ui/src/app/applications/components/application-details/application-resource-list.tsx

```diff
@@ -12,82 +12,98 @@ export const ApplicationResourceList = ({
     onNodeClick,
     nodeMenu
 }: {
-    resources: models.ResourceStatus[];
+    resources: any[];
```

Comment:
i think it is not good idea change it to any, can you please create new type if needed

Reply:
I agree, it turned out I already added the optional field in the `models.ResourceStatus` type. So I can keep it as it was. Thanks for the catch ^^